src/Sound.ts

```diff
@@ -111,112 +103,91 @@ export class Sound
-     * @private
      */
     private _muted: boolean;
 
     /**
      * The internal volume.
-     * @name PIXI.sound.Sound#_loop
      * @type {boolean}
-     * @private
      */
     private _loop: boolean;
 
     /**
      * The internal playbackRate
-     * @name PIXI.sound.Sound#_speed
      * @type {number}
-     * @private
      */
     private _speed: number;
 
     /**
      * Create a new sound instance from source.
-     * @method PIXI.sound.Sound.from
-     * @param {ArrayBuffer|String|Object|HTMLAudioElement} options Either the path or url to the source file.
+     * @param {ArrayBuffer|String|Object|HTMLAudioElement} options - Either the path or url to the source file.
      *        or the object of options to use.
-     * @param {String} [options.url] If `options` is an object, the source of file.
-     * @param {HTMLAudioElement|ArrayBuffer} [options.source] The source, if already preloaded.
-     * @param {boolean} [options.autoPlay=false] true to play after loading.
-     * @param {boolean} [options.preload=false] true to immediately start preloading.
-     * @param {boolean} [options.singleInstance=false] `true` to disallow playing multiple layered instances at once.
-     * @param {number} [options.volume=1] The amount of volume 1 = 100%.
-     * @param {number} [options.speed=1] The playback rate where 1 is 100% speed.
-     * @param {Object} [options.sprites] The map of sprite data. Where a sprite is an object
+     * @param {String} [options.url] - If `options` is an object, the source of file.
+     * @param {HTMLAudioElement|ArrayBuffer} [options.source] - The source, if already preloaded.
+     * @param {boolean} [options.autoPlay=false] - true to play after loading.
+     * @param {boolean} [options.preload=false] - true to immediately start preloading.
+     * @param {boolean} [options.singleInstance=false] - `true` to disallow playing multiple layered instances at once.
+     * @param {number} [options.volume=1] - The amount of volume 1 = 100%.
+     * @param {number} [options.speed=1] - The playback rate where 1 is 100% speed.
+     * @param {Object} [options.sprites] - The map of sprite data. Where a sprite is an object
      *        with a `start` and `end`, which are the times in seconds. Optionally, can include
      *        a `speed` amount where 1 is 100% speed.
-     * @param {PIXI.sound.Sound~completeCallback} [options.complete=null] Global complete callback
+     * @param {Sound~completeCallback} [options.complete=null] - Global complete callback
      *        when play is finished.
-     * @param {PIXI.sound.Sound~loadedCallback} [options.loaded=null] Call when finished loading.
-     * @param {boolean} [options.loop=false] true to loop the audio playback.
-     * @return {PIXI.sound.Sound} Created sound instance.
+     * @param {Sound~loadedCallback} [options.loaded=null] - Call when finished loading.
+     * @param {boolean} [options.loop=false] - true to loop the audio playback.
```

Comment:
Would it be possible to move the options documentation to the Options interface (and drop the types/names)?

Reply:
Yup, good call